index.ts

```diff
@@ -325,6 +325,9 @@ collect.set('isRepoRoot', [
 	'https://github.com/sindresorhus/refined-github/tree/master?files=1',
 ]);
 
+export const isConversation = (url: URL | Location = location): boolean => isIssue(url) || isPRConversation(url);
+collect.set('isConversation', combinedTestOnly);
+
```

Comment:
Why in this specific spot? It randomly between isRepo* tests. It should either be alphabetical or near a similar one (but alphabetical in this case)

Reply:
Because I did it from my tablet :)